### ISSUE TITLE
Problems with InertiaHead component on Vue 3

### ISSUE THREAD

**Issue by d8vjork** [inertiajs/inertia#715]
### Versions:

- `@inertiajs/inertia` version: 0.9.1
- `@inertiajs/inertia-vue3` version: 0.4.2

### Describe the problem:

Problem using InertiaHead global component on my Vue 3 app:

`<inertia-head title="Test - MyApp" />`

![image](https://user-images.githubusercontent.com/2331052/120626778-4a29ec80-c463-11eb-830b-8e918004f04f.png)

### Steps to reproduce:

Just use `<inertia-head>` component on another

**Comment by emargareten** [comment]
remove the `:` from `:title="Test - MyApp"`

**Comment by d8vjork** [comment]
@emargareten Sorry, that was already a typo (if you see the screenshot doesn't have it)

**Comment by d8vjork** [comment]
I think the problem is around here: https://github.com/inertiajs/inertia/blob/3948b433040a2205d6b1d8ea2262728cc62714c4/packages/inertia-vue3/src/head.js#L10

My question is, is this working for any of you guys?

**Comment by reinink** [comment]
Are you **certain** you're on the latest versions?

**Comment by d8vjork** [comment]
Ok close this, actually is totally my fault, didn't notice that Inertia's is adding properties to the Vue's **config.globalProperties** and I was replacing all of them on my own app

Sorry :P

**Comment by d8vjork** [comment]
@reinink Ok, sorry again, but there's still problems with this:

![image](https://user-images.githubusercontent.com/2331052/120781607-832c9480-c529-11eb-9b9b-bbcc3efc56c6.png)

I'm using the [extending example](https://inertiajs.com/title-and-meta#extending), and apparently fails whenever we've slot content sent to InertiaHead component

**Comment by reinink** [comment]
Hello @d8vjork, please post the code you're writing to generate this. Just having the error isn't enough.

**Comment by d8vjork** [comment]
Here it is (is basically the example of the documentation):

### resources/js/app.js

```javascript
import { createInertiaApp } from "@inertiajs/inertia-vue3";
import { createApp, h } from "vue";
import { AppPlugin } from "./plugin";
import AppShell from "@/Components/Common/AppShell";

createInertiaApp({
  resolve: (name) =>
    import(/* webpackChunkName: "Pages/[request]" */ `./Pages/${name}`).then(
      ({ default: page }) => {
        console.time(`Page ${page.name} rebuilding`);

        page.layout = [].concat(page.layout);

        if (page.layout.indexOf(AppShell) === -1) {
          page.layout = page.layout.concat(AppShell).filter(Boolean);
        }

        console.timeEnd(`Page ${page.name} rebuilding`);

        return page;
      }
    ),
  setup({ el, app, props, plugin }) {
    createApp({ render: () => h(app, props) })
      .use(plugin)
      .use(AppPlugin)
      .mount(el);
  },
});
```

### AppHead.vue

```javascript
<template>
  <inertia-head :title="title ? `${title} - Skore` : 'Skore'">
    <slot />
  </inertia-head>
</template>

<script>
export default {
  props: {
    title: {
      type: String,
      default: "",
    },
  },
}
</script>
```

### Page.vue

```javascript
<template>
    <AppHead :title="pageTitle" />
    <!-- .... -->
</template>

<script>
import AppHead from "@/Components/Common/AppHead";

export default {
  name: "MyPage",

  components: {
    AppHead,
  },
};
</script>

```

**Comment by reinink** [comment]
Can you show me the content from `<inertia-head>` in your page component? I think that's the issue.

**Comment by d8vjork** [comment]
@reinink It has no content, is exactly the same as the one in the documentation, even having a meta as content of that AppHead it gives me the exact same error

**Comment by reinink** [comment]
Sorry, I mean can I see the content of your `<app-head>` on your page?

**Comment by d8vjork** [comment]
@reinink Is the second code block on the previous comment

**Comment by reinink** [comment]
Is that the only instance of `<app-head>` in your app?

**Comment by d8vjork** [comment]
Yes, I only have it on the layout I'm using for my app

**Comment by reinink** [comment]
Gotcha. I am able to reproduce this. Leave it with me. 👍

**Comment by reinink** [comment]
Fixed and released: `@inertiajs/inertia-vue3@0.4.3`
